newIDE/app/src/AssetStore/ExtensionStore/ExtensionInstallDialog.js

```diff
@@ -141,6 +141,9 @@ export default class ExtensionInstallDialog extends Component<Props, State> {
               <Text noMargin size="title">
                 {extensionShortHeader.fullName}
               </Text>
+              <Text noMargin size="body2">
+                <Trans>Version:</Trans> {' ' + extensionShortHeader.version}
```

Comment:
You an actually put he { } inside the Trans tag, as this will create a placeholder so that language so that have a different order can translate things in the proper order:

```suggestion
                <Trans>Version {' ' + extensionShortHeader.version}</Trans>
```